packages/global-styles/src/hooks/use-color-palette-variations.ts

```diff
@@ -18,7 +18,10 @@ const useColorPaletteVariations = (
 				path: `/sites/${ encodeURIComponent( siteId ) }/global-styles-variation/color-palettes`,
 				method: 'GET',
 				apiNamespace: 'wpcom/v2',
-				query: new URLSearchParams( { stylesheet } ).toString(),
+				query: new URLSearchParams( {
+					stylesheet,
+					base_variation_stylesheet: 'pub/assembler',
```

Comment:
This is for testing. If everything works well, we will make changes on the endpoint and remove this.